### PR TITLE
chromium page.url tests

### LINKED ISSUE #72: Pass page.spec.js tests in Chromium
Let's turn these tests green, let's remember to check Puppeteer's code

### PR DESCRIPTION
**Pull request by Meir017**
closes #72

### CODE REVIEW

File: src/PlaywrightSharp.Chromium/ChromiumBrowserApp.cs [hunk 1]
```diff
@@ -10,10 +10,12 @@ namespace PlaywrightSharp.Chromium
     public class ChromiumBrowserApp : IBrowserApp
```
Comment (kblok) [inline src/PlaywrightSharp.Chromium/ChromiumBrowserApp.cs]: FYI. My next task is making this class a generic class. There is not `crBrowserApp` un Playwright.

File: src/PlaywrightSharp.Chromium/ChromiumSession.cs [hunk 2]
```diff
@@ -73,9 +72,16 @@ internal async Task<TChromiumResponse> SendAsync<TChromiumResponse>(IChromiumReq
             return (TChromiumResponse)result;
         }
 
-        internal void Close(string reason)
+        internal void OnClosed()
```
Comment (kblok) [inline src/PlaywrightSharp.Chromium/ChromiumSession.cs]: let's keep the reason and add that to the exception message.

File: src/PlaywrightSharp/Page.cs [hunk 3]
```diff
@@ -496,6 +514,19 @@ internal void OnPopup(object popupPage)
 
         internal void DidDisconnected()
         {
+            _disconnected = true;
```
Comment (kblok) [inline src/PlaywrightSharp/Page.cs]: arrow

**Comment by Meir017** [comment]
@kblok the next stage is the evaluate ability

**Comment by kblok** [comment]
> @kblok the next stage is the evaluate ability

Yeah, I'll end up covering that for #68

**Pull request merged**